pkg/grpc/device_definition.proto

```diff
@@ -95,42 +73,37 @@ message GetDeviceDefinitionByMMYRequest {
 }
 
 message GetIntegrationResponse {
-	repeated GetIntegrationItemResponse integrations = 1;
+	repeated DeviceIntegration integrations = 1;
 }
 
 message EmptyRequest
```

Comment:
[`google.protobuf.Empty`](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Empty) might interest you.